### PR TITLE
use cloudevent client from ocm api.

### PR DESCRIPTION
**Pull request by morvencao**

### CODE REVIEW

File: internal/cloudevents/payload.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package cloudevents
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (qiujian16) [inline internal/cloudevents/payload.go:4]: I think the resource definition should be in the api repo finally?

File: internal/cloudevents/resource.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+func (resLister *ResourceLister) List(listOpts types.ListOptions) ([]*db.Resource, error) {
+	resources, err := db.ListResourceByConsumer(listOpts.ClusterName)
+	if err != nil {
+		return nil, err
+	}
+	return resources, nil
+}
+
+func ResourceStatusHashGetter(obj *db.Resource) (string, error) {
```
Comment (qiujian16) [inline internal/cloudevents/resource.go:25]: similar, we should have a predefined status hash in api repo

File: internal/cloudevents/payload.go (hunk 1 above)
Comment (morvencao) [reply internal/cloudevents/payload.go:4]: @skeeey will create a directory in api repo holds the previously mentioned manifest and associated payload, as per our earlier conversation.

**Pull request merged**